### ISSUE TITLE
Allow kwargs to be passed to export() call

### ISSUE THREAD

**Issue by lcadalzo** [twosixlabs/armory#1399]
[This](https://github.com/twosixlabs/armory/pull/1297/commits/00f9b761e560b489b144b6fa16a1f6ace47b689a) commit removed the ability to pass extra kwargs to the `export()` call. While this commit simplified the scenario code as it pertains to exporters, it had this unintended side effect.

Calling the method as such (copied from `exporting_data.md`), for example, will now fail with an unexpected kwarg:

```
>>> s.sample_exporter.export(
                      x=s.x, 
                      x_adv=s.x_adv, 
                      y=s.y, 
                      y_pred_clean=s.y_pred, 
                      y_pred_adv=s.y_pred_adv,
                      with_boxes=True)
```

**Comment by davidslater** [comment]
Can this be incorporated back into Sterling's PR?

**Comment by lcadalzo** [comment]
@swsuggs I can push a commit to your branch with a proposed fix

**Comment by lcadalzo** [comment]
Fixed in #1398
